lib/xlibc/include/threads.h

```diff
@@ -75,10 +75,10 @@ int thrd_join (thrd_t thr, int *res);
 int thrd_sleep (const struct timespec *duration, struct timespec *remaining);
 void thrd_yield (void);
 
-int tss_create (tss_t *key, tss_dtor_t dtor) __unimplemented;
-void tss_delete (tss_t key) __unimplemented;
-void *tss_get (tss_t key) __unimplemented;
-int tss_set (tss_t key, void *val) __unimplemented;
+int tss_create (tss_t *key, tss_dtor_t dtor);
+void tss_delete (tss_t key);
+void *tss_get (tss_t key);
+int tss_set (tss_t key, void *val);
```

Comment:
Did you consider implementing a standard API for this instead? I think using win32 function names / signatures would make a lot of sense here.